<?xml version='1.0' encoding='utf-8'?>

<launch>
    <!--打开语音唤醒-->
    <include file="$(find castlex_voice_system)/launch/castlex_awake.launch"/>

    <!--打开命令词识别-->
    <node pkg = "castlex_voice_system" type = "castlex_auto_order_node" name = "castlex_auto_order_node" output="screen"/>

    <!--打开命令词解析-->
    <node pkg = "castlex_voice_system" type = "castlex_multiword.py" name = "XML_Analysis" output="screen">
        <!-- 设置命令词识别的打印结果的模板 -->
        <param name="file_path" value="$(find castlex_voice_system)/params/castlex_auto.xml" />
        <!--     设置识别完之后的提示音 -->
        <param name="failed_file_path" value="$(find castlex_voice_system)/params/voice/failed.mp3" />
        <param name="Received_file_path" value="$(find castlex_voice_system)/params/voice/Received.mp3" />
        <param name="ReEnterAuido_file_path" value="$(find castlex_voice_system)/params/voice/ReEnterAuido.mp3" />      
    </node>

</launch>
